src/Interface/Modules/Render/ViewScene.cc

```diff
@@ -1881,7 +1939,7 @@ void ViewSceneDialog::setSpecularValue(double value)
 void ViewSceneDialog::setShininessValue(double value)
 {
   state_->setValue(Modules::Render::ViewScene::Shine, value);
-  setMaterialFactor(SRInterface::MAT_SHINE, value);
+  setMaterialFactor(SRInterface::MAT_SHINE, value * value * 39 + 1);
```

Comment:
Refactor magic number usage--make a function (even a lambda would be an improvement)